examples/example.c

```diff
@@ -4,8 +4,9 @@
 #include <string.h>
 
 static void
-print_envelope(sentry_envelope_t *envelope, void *_data)
+print_envelope(sentry_envelope_t *envelope, void *unused_data)
```

Comment:
Can you use `UNUSED` here?

Reply:
The example only uses public API, which `UNUSED` isn’t…